applications-frontend/src/components/Applicant.vue

```diff
@@ -1,61 +1,83 @@
 <template>
-    <div class="container">
-        <div class="columns">
-            <aside class="review column is-one-quarter">
-                <form>
-                    <div class="field">
-                        <label class="label is-large">Your Score: </label>
-                        <div class="control is-large">
-                            <input class="input" type="number" :min="min" :max="max"/>
-                        </div>
-                    </div>
-                </form>
-            </aside>
-
-            <div class="column is-three-quarters">
-                <!-- random demographic information, can find a nice way to represent or table it -->
-                <section class="basicinfo">
-                    <h1 class="title">John Wick</h1>
-                    <h2 class="subtitle">Baba Yaga</h2>
-                </section>
-                <!-- responses go here -->
-                <section class="responses">
-                
-                </section>
-
-                <!-- random web links can go here -->
-                <section class="links">
-
-                </section>
-            </div>
-        </div>
-    </div>
+<div class="container">
+	<div class="columns">
+
+		<!-- Control panel -->
+		<div class="column has-background-light is-3">		
+			<center>
+				<!-- Person's Image -->
+				<figure class="image is-128x128">
+					<img class="profile-picture" alt="" src="https://bulma.io/images/placeholders/256x256.png">
+				</figure>
+
+				<!-- Person's name -->			
+				<div class="subtitle is-4" style="margin: 16px;">Full-name</div>
+
+				<!-- Break for looking good :D -->
+				<div class="pad-light"></div>
+				<div class="break"></div>
+				<div class="pad-light"></div>
+
+				<!-- Rating control -->
+				<div class="subtitle is-4" style="margin: 16px;">Rating</div>
+				<div class="buttons has-addons is-centered">
+					<span class="button">1</span>
+					<span class="button">2</span>
+					<span class="button">3</span>
+					<span class="button">4</span>
+					<span class="button">5</span>
+				</div>
+
+				<!-- Next applicant button -->
+				<a class="button is-primary is-medium">Next applicant</a>
+				<div class="pad"></div>			
+			</center>		
+		</div>
+
+		<!-- Information panel -->
+		<div class="column has-background-light">
+
+			<!-- Major -->
+			<div class="title" style="margin: 16px;">Major</div>	
+			<div class="subtitle is-4" style="margin: 16px;">{{major}}</div>
+			<div class="pad-light"></div>		
+
+			<!-- Year -->
+			<div class="title" style="margin: 16px;">College year</div>	
+			<div class="subtitle is-4" style="margin: 16px;">{{year}}</div>	
+			<div class="pad-light"></div>	
+
+			<!-- Essays -->
+			<div class="title" style="margin: 16px;">Text information</div>	
+			<div class="subtitle is-4" style="margin: 16px;">REPLACE ME</div>	
```

Comment:
You're going to have to use v-for to insert a series of essay responses here, in a future commit